vllm/v1/worker/gpu_model_runner.py

```diff
@@ -136,7 +136,13 @@ def __init__(
         self.num_sms = self.device_properties.multi_processor_count
 
         # Persistent buffers for CUDA graphs.
-        self.input_ids = torch.zeros(self.max_num_tokens,
+        # Sizing `input_ids` to `max_num_tokens + 1` allows
```

Comment:
I don't think this is needed:
- if we have a partial prompt, it will always be of size < `max_num_tokens`